addons/mail/static/src/components/message_context_menu/message_context_menu.js

```diff
@@ -0,0 +1,32 @@
+/** @odoo-module **/
+
+import { useComponentToModel } from '@mail/component_hooks/use_component_to_model';
+import { registerMessagingComponent } from '@mail/utils/messaging_component';
+
+const { Component } = owl;
+
+export class MessageContextMenu extends Component {
+
+    /**
+     * @override
+     */
+     setup() {
+        super.setup();
+        useComponentToModel({ fieldName: 'component' });
```

Comment:
It's doesn't look like you need this `component` field. Better not introduce it if you don't need it.
This comment applies to some other newly introduced components.